README-DEV.rst

```diff
@@ -64,6 +64,21 @@ associated ``devcontainer.json`` file to quickly provision a
 development environment using `GitHub Codespaces <https://github.com/features/codespaces>`_
 or `Visual Studio Code <https://code.visualstudio.com/docs/remote/containers>`_.
 
+
+.. image:: https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode
+    :target: https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/apache/couchdb
+
+If you already have VS Code and Docker installed, you can click the badge above or 
+`here <https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/apache/couchdb>`_ 
+to get started. Clicking these links will cause VS Code to automatically install the 
+Remote - Containers extension if needed, clone the source code into a container volume, 
+and spin up a dev container for use.
+
+This ``devcontainer`` will automatically run ``./configure && make`` the first time it is created.  
+While this may take some extra time to spin up, this tradeoff means you will be able to 
+run things like ``./dev/run`` and ``make check`` straight away.  Subsequent startups should be quick.
```

Comment:
I think we need to give `./dev/run` and admin user with password or the switch `--with-admin-party-please`. Do you tried it with only `./dev/run`?

Reply:
It works with all of the above, I've tried them all.  Would you like this to be updated to show all three examples? Or just one of them?